packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -31,7 +31,7 @@ import {
   ContextProvider,
   ForwardRef,
   Block,
-} from 'react-reconciler/src/ReactWorkTags';
+} from 'shared/ReactWorkTags';
```

Comment:
Same.

Reply:
They are both types/const-only. There is no implementation code in either of those files. Let me know if you want an even slimmer PR with literal types-only files, I can undo these type/const-only moves. Thanks for reviewing!

Reply:
`ReactWorkTags` exports constants, doesn't it? That's why we see `import` here rather than `import type`.

Reply:
done, thanks for the feedback!